test/integration/daemonset/daemonset_test.go

```diff
@@ -105,6 +106,7 @@ func setupScheduler(
 
 	defaultProviderName := schedulerconfig.SchedulerDefaultProviderName
 
+	ctx := context.Background()
```

Comment:
We can replace the pass-in parameter `stopCh chan struct{}` to a context.Context, so we don't need to construct another ctx here.